Header header

# дистанция назад, [м], [0, inf)
float32 downward